internal/cmd/api-key/command_delete.go

```diff
@@ -10,13 +10,15 @@ import (
 	"github.com/confluentinc/cli/internal/pkg/form"
 	"github.com/confluentinc/cli/internal/pkg/output"
 	"github.com/confluentinc/cli/internal/pkg/resource"
+	"github.com/confluentinc/cli/internal/pkg/types"
+	"github.com/confluentinc/cli/internal/pkg/utils"
 )
 
 func (c *command) newDeleteCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:               "delete <api-key>",
-		Short:             "Delete an API key.",
-		Args:              cobra.ExactArgs(1),
+		Use:               "delete <api-key-1> [api-key-2] ... [api-key-N]",
```

Comment:
Nit: We use lower-kebab-case for all of our commands and flags, so we should probably do the same for args.
```suggestion
		Use:               "delete <api-key-1> [api-key-2] ... [api-key-n]",
```